tests/Fixer/Import/NoUnusedImportsFixerTest.php

```diff
@@ -679,6 +679,62 @@ public function testWithSameNamespaceImportAndUnusedImportAfterNamespaceStatemen
 {
 }
 
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testFunctionsInTheGlobalNamespaceShouldNotBeRemoved()
+    {
+        $expected = <<<'EOF'
+<?php
+
+namespace Foo;
+
+use function is_int;
+
+is_int(1);
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+
+namespace Foo;
+
+use function is_int;
+use function is_float;
+
+is_int(1);
+
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testConstantsInTheGlobalNamespaceShouldNotBeRemoved()
+    {
+        $expected = <<<'EOF'
+<?php
+
+namespace Foo;
+
+use function PHP_INT_MAX;
+
+echo PHP_INT_MAX;
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+
+namespace Foo;
+
+use function PHP_INT_MAX;
+use function PHP_INT_MIN;
```

Comment:
I think you meant `use const PHP_INT_MAX;` ?  thanks for adding support for this :+1: